lib/publiccloud/azure.pm

```diff
@@ -203,8 +235,7 @@ sub get_image_name {
     my ($self, $file) = @_;
     my ($img_name) = $file =~ /([^\/]+)$/;
     $img_name =~ s/\.xz$//;
-    my $gen = (check_var('PUBLIC_CLOUD_AZURE_SKU', 'gen2')) ? 'V2' : 'V1';
-    $img_name =~ s/\.vhdfixed/-$gen.vhd/;
+    $img_name =~ s/\.vhdfixed/.vhd/;
```

Comment:
Is it intentional to remove the `$gen` from `$img_name`? :thinking:

Reply:
Yes. I think it's not needed. Our images are universal and it makes no point to differ the blob name from the original image name. Or?

Reply:
Right, I'm more concerned about an unintended name collision within the Azure galleries. If the VRs include a test run of the same image running both `gen1` and `gen2` HyperV, then we can check if this matters or not.

Reply:
Of course it will 👍